gatsby/src/hooks/useAnalytics.ts

```diff
@@ -0,0 +1,6 @@
+import React from "react"
+
+export const useAnalytics = () => {
+  const [analyticsInstance] = React.useState(window.appInsights)
```

Comment:
no tady vidi

Reply:
mby protože je to v src složce, zkus dát ty typy do rootu gatsby

Reply:
zkusil jsem to, ale situace se změnila opačným směrem. Konfigurace vidí, ale komponenta ne